### DISCUSSION TITLE
Monitor Ubiquiti NVR

### DISCUSSION THREAD

**Discussion by bguenther2021** [amidaware/tacticalrmm#772] [General]
Hi,

Thanks for this awesome project. I have a client with a ubiquiti NVR so i cannot install an Agent on it. Is there anyway i can setup a monitor that will alert me if that computer goes down? And maybe also monitor an IP camera.

Regards,
Ben

**Comment by dinger1986** [comment]
Setup a ping check?

**Reply by bguenther2021** [reply]
How do i add an agent when not using the installer?

**Reply by dinger1986** [reply]
You can't. Install the agent on another machine on the same network and setup a ping check to monitor it all

**Reply by bguenther2021** [reply]
Thanks, simple enough.
